src/core/kprobe/v0_4_0.rs

```diff
@@ -23,9 +23,9 @@ pub struct Kprobe {
 }
 
 impl Kprobe {
-    fn new(name: &str, attach_type: u32, function: &str, code: File) -> Result<Self, Error> {
+    fn new(name: &str, attach_type: u32, function: &str, code: File) -> Result<Self> {
         let cname =
-            CString::new(name).map_err(|_| format_err!("Nul byte in Kprobe name: {}", name))?;
+            CString::new(name).map_err(|_| anyhow::anyhow!("Nul byte in Kprobe name: {}", name))?;
```

Comment:
it looks like anyhow exports `format_err!`? If so, that could save a lot of lines in this diff by doing `use anyhow::*;`